src/worker/plugins/run.ts

```diff
@@ -44,7 +44,8 @@ export async function runOnSnapshot(server: PluginsServer, event: PluginEvent):
 }
 
 export async function runProcessEvent(server: PluginsServer, event: PluginEvent): Promise<PluginEvent | null> {
-    const pluginsToRun = getPluginsForTeam(server, event.team_id)
+    const team_id = event.team_id
```

Comment:
Nit: I'd follow JS `camelCase` convention even if the prop uses `snake_case`

Reply:
Argh, how did I not notice everything else using JS `camelCase` lol. A python habit xD - is there a specific style guide we're following?